lib/bitcrowd_ecto/changeset.ex

```diff
@@ -316,4 +317,106 @@ defmodule BitcrowdEcto.Changeset do
       changeset
     end
   end
+
+  @doc "Validates the currency of the given money field"
+  @doc since: "0.13.0"
+  @spec validate_currency(Changeset.t(), atom(), atom()) :: Changeset.t()
+  def validate_currency(changeset, field, required_currency \\ :EUR) do
+    validate_change(
+      changeset,
+      field,
+      :valid_currency,
+      fn _, %Money{currency: currency} ->
+        if currency == required_currency do
+          []
+        else
+          [
+            {field, {"invalid currency", [validation: :validate_currency, fields: [field]]}}
```

Comment:
* Why is this `fields: [field]` option needed here? It's not present in the other functions.
* Also, the `validation` option for `validate_foo` is usually `:foo`, see the other functions.
* We could give the `required_currency` as a metadatum though? Wdyt?

```suggestion
            {field, {"invalid currency", [validation: :currency, required_currency: required_currency]}}
```